src/runtime/vdom/vdom-render.ts

```diff
@@ -135,9 +135,10 @@ const createElm = (oldParentVNode: d.VNode, newParentVNode: d.VNode, childIndex:
     }
   }
 
+  // This needs to always happen so we can hide nodes that are projected through
+  // to another component but don't end up in a slot
```

Comment:
nit, non-blocking suggestion
```suggestion
  // This needs to always happen so we can hide nodes that are projected
  // to another component but don't end up in a slot
```